Light up bot/lightupbot.py

```diff
@@ -23,53 +23,29 @@ async def connect():
     )
     return await RobotClient.at_address(robot_address, opts)
 
-#for displaying your detections with a red box around them, if needed
-def drawBox(d, frame):
-  rect = [d.x_min, d.y_min, d.x_max, d.y_max]
-  img1 = ImageDraw.Draw(frame)
-  img1.rectangle(rect, outline="red")
-  return frame
-
 async def main():
     robot = await connect()
 
-    print('Resources:')
-    print(robot.resource_names)
     # This string should match your camera component name in your robot config on the Viam app
     camera = Camera.from_robot(robot, "my-camera")
```

Comment:
This line does not need to be here any more,  since you are not getting or using images from it in this script